### PR TITLE
Stored response in variable

### PR DESCRIPTION
**Pull request by JinaliHShah**

### CODE REVIEW

File: src/main/java/com/cohortsys/controlstation/enums/Command.java [hunk 1]
```diff
@@ -11,7 +11,8 @@
     PAN("0"),
     TILT("0"),
     DETECT("distance"),
-    BRAKE ("break");
+    //changed
+    BRAKE ("BRAKE");
```
Comment (JeremyJames5x5) [inline src/main/java/com/cohortsys/controlstation/enums/Command.java]: Why was this changed?

File: src/main/java/com/cohortsys/reasoning/AgentAutonomous.java [hunk 2]
```diff
@@ -57,7 +57,13 @@ public String getAgName() {
     @Override
     public List<Literal> perceive() {
         List<Literal> l = new ArrayList<>(brain.getPerceptions());
-        return l;
+        //change here
+        if(l.isEmpty()){
+         return (List<Literal>) super.perceive();
```
Comment (JeremyJames5x5) [inline src/main/java/com/cohortsys/reasoning/AgentAutonomous.java]: Where is the super class getting its perceptions from?

File: src/main/java/com/cohortsys/reasoning/AgentAutonomous.java [hunk 3]
```diff
@@ -57,7 +57,13 @@ public String getAgName() {
...
+        }
+        else {
```
Comment (JeremyJames5x5) [inline src/main/java/com/cohortsys/reasoning/AgentAutonomous.java]: Format your braces properly

File: src/main/java/com/cohortsys/reasoning/Brain.java [hunk 4]
```diff
@@ -82,7 +83,14 @@ public void run() {
                 }
 
                 if (isNewActionPerformed) {
-                    updatePerception(response);
+                    //change here
```
Comment (JeremyJames5x5) [inline src/main/java/com/cohortsys/reasoning/Brain.java]: No need to put "change here", just write a proper comment.

File: src/main/java/com/cohortsys/reasoning/Brain.java [hunk 5]
```diff
@@ -82,7 +83,14 @@ public void run() {
...
+                    if(response.equals("drive successful") || response.equals("steer successful") || response.equals("brake successful")) {
+                        updatePerception("get_distance");
+                    } else {
+                        double num = Double.parseDouble(response);
+                        updatePerception("eval_distance(" + num + ")");
+                    }
+//                    updatePerception(response);
```
Comment (JeremyJames5x5) [inline src/main/java/com/cohortsys/reasoning/Brain.java]: remove if no more in use

File: src/main/java/com/cohortsys/reasoning/Brain.java [hunk 6]
```diff
@@ -41,7 +41,7 @@ public Brain(StationClient client, boolean isAuto, String name, String aslFilePa
     public void run() {
 
         String response;
-        String distance;
+//        String distance;
```
Comment (JeremyJames5x5) [inline src/main/java/com/cohortsys/reasoning/Brain.java]: remove if no more in use

File: src/main/java/com/cohortsys/reasoning/AgentAutonomous.java [hunk 7]
```diff
@@ -57,7 +57,13 @@ public String getAgName() {
     @Override
     public List<Literal> perceive() {
         List<Literal> l = new ArrayList<>(brain.getPerceptions());
-        return l;
+        //change here
```
Comment (JeremyJames5x5) [inline src/main/java/com/cohortsys/reasoning/AgentAutonomous.java]: use proper comment instead

File: src/main/java/com/cohortsys/controlstation/enums/Command.java [hunk 8]
```diff
@@ -24,11 +25,20 @@ public String getValue() {
...
 
     public static Command parseCommand(String command){
+        //changed here for drive_25_25
+        command=command.replaceAll("sub","-");
         String[] cmd = command.split("_");
         Command com = BRAKE;
         if (cmd.length != 0){
             try {
-                com = Command.valueOf(cmd[0].toUpperCase()).setValue(cmd[1]);
+                System.out.println(cmd.length);
```
Comment (JeremyJames5x5) [inline src/main/java/com/cohortsys/controlstation/enums/Command.java]: Remove sout, we will use a logger instead.

File: src/main/java/com/cohortsys/controlstation/enums/Command.java (hunk 1 above)
Comment (JinaliHShah) [reply src/main/java/com/cohortsys/controlstation/enums/Command.java]: Spelling mistake

File: src/main/java/com/cohortsys/reasoning/AgentAutonomous.java (hunk 2 above)
Comment (JinaliHShah) [reply src/main/java/com/cohortsys/reasoning/AgentAutonomous.java]: Taking perception from belief base

File: src/main/java/com/cohortsys/reasoning/Brain.java (hunk 4 above)
Comment (JinaliHShah) [reply src/main/java/com/cohortsys/reasoning/Brain.java]: Done

File: src/main/java/com/cohortsys/controlstation/enums/Command.java (hunk 8 above)
Comment (JinaliHShah) [reply src/main/java/com/cohortsys/controlstation/enums/Command.java]: Done
